### PR TITLE
[FL-2047] SubGhz: New GUI ReadRAW view

### PR DESCRIPTION
**Pull request by Skorpionm**
# What's new

- [FL-2047] SubGhz: New GUI RAW Read view
- SubGhz: fix bug wrong frequency of the allowed transmission

# Verification 

- Compile and download the firmware
- Download the Sub Ghz Read RAW application, and check if it works

# Checklist (do not modify)

- [X] PR has description of feature/bug or link to Confluence/Jira task
- [X] Description contains actions to verify feature/bugfix
- [X] I've built this code, uploaded it to the device and verified feature/bugfix

[FL-2047]: https://flipperzero.atlassian.net/browse/FL-2047?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by skotopes — APPROVED** [review]

File: applications/subghz/views/subghz_read_raw.c [hunk 1]
```diff
@@ -89,6 +83,90 @@ void subghz_read_raw_update_sample_write(SubghzReadRAW* instance, size_t sample)
...
+            32 + subghz_read_raw_tab_sin((i + model->ind_sin * 16 + 1) * 2) /
+                     SUBGHZ_RAW_SIN_AMPLITUDE);
+        canvas_draw_line(
+            canvas,
+            i + 1,
+            32 - subghz_read_raw_tab_sin((i + model->ind_sin * 16)) / SUBGHZ_RAW_SIN_AMPLITUDE,
+            i + 2,
+            32 + subghz_read_raw_tab_sin((i + model->ind_sin * 16 + 1) * 2) /
+                     SUBGHZ_RAW_SIN_AMPLITUDE);
+        // canvas_draw_line(
```
Comment (skotopes) [inline applications/subghz/views/subghz_read_raw.c]: remove

File: applications/subghz/views/subghz_read_raw.c [hunk 2]
```diff
@@ -223,10 +346,9 @@ void subghz_read_raw_exit(void* context) {
                 instance->callback(SubghzCustomEventViewReadRAWIDLE, instance->context);
                 model->satus = SubghzReadRAWStatusStart;
             }
-            string_reset(model->frequency_str);
-            string_reset(model->preset_str);
-            string_reset(model->sample_write);
-            free(model->rssi_history);
+            //string_reset(model->frequency_str);
```
Comment (skotopes) [inline applications/subghz/views/subghz_read_raw.c]: remove

**Review by skotopes — CHANGES_REQUESTED** [review]

File: applications/subghz/views/subghz_read_raw.c (hunk 1 above)
Comment (Skorpionm) [reply applications/subghz/views/subghz_read_raw.c]: ok

File: applications/subghz/views/subghz_read_raw.c (hunk 2 above)
Comment (Skorpionm) [reply applications/subghz/views/subghz_read_raw.c]: ok

**Review by skotopes — APPROVED** [review]

**Pull request merged**